1-server-side/msal-webapi-ciam-sample/msal-web-api/src/main/java/com/microsoft/azure/msalapiciamsample/AADClaimsVerifier.java

```diff
@@ -0,0 +1,67 @@
+package com.microsoft.azure.msalapiciamsample;
+
+import org.springframework.security.oauth2.common.exceptions.InvalidTokenException;
+import org.springframework.security.oauth2.provider.token.store.JwtClaimsSetVerifier;
+import org.springframework.util.Assert;
+import org.springframework.util.CollectionUtils;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Map;
+
+public class AADClaimsVerifier implements JwtClaimsSetVerifier {
+    private static final String ISS_CLAIM = "iss";
+    private static final String AUD_CLAIM = "aud";
+    private static final String V2_ISSUER_FORMAT = "https://%s.ciamlogin.com/%s/v2.0";
+
+
+    private HashSet<String> acceptedIssuers = new HashSet<>();
+    private final String applicationId;
+
+    public AADClaimsVerifier(final String[] acceptedTenants, final String applicationId) {
```

Comment:
Q: does the other web api sample have the same concept of "accepted tenants" i.e. multiple tenants? If in doubt, I think single tenant web api is sufficient for this sample, no need to complicate it.

Reply:
Yeah, the other API sample that I used as the base for this one allowed multi-tenant apps. In the latest commit, I changed the verifier/some variable wording to only focus on a single tenant.